sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -154,7 +161,7 @@ func App() *cli.App {
 	return app
 }
 
-func runCtr(containerdSocket string, namespace string, containerID string, source string, superpowered bool) error {
+func runCtr(containerdSocket string, namespace string, containerID string, source string, superpowered bool, bootstrap bool) error {
```

Comment:
Since this is a private function that really only gets called in one place, I don't know if we need to do that yet.  One change we could make is to separate this function a bit more.  The first two arguments here are only used up until line 181 for constructing the client; if we extract the client object from this function and instead pass it in (either directly or as a data field on a receiver object) that might be a bit cleaner.  I think that's the route I would go.